source/postprocess/dynamic_topography.cc

```diff
@@ -247,34 +243,24 @@ namespace aspect
             // is true and will be changed to false if it's at the lower boundary. If the
             // cell is at neither boundary the loop will continue to the next cell.
             bool at_upper_surface = true;
-            for (unsigned int f=0; f<GeometryInfo<dim>::faces_per_cell; ++f)
+            for (unsigned int f=0; f<GeometryInfo<3>::faces_per_cell; ++f)
```

Comment:
Why 3? Isnt this class allowed in 2d?

Reply:
Ah, thanks for catching that! I brought this over from the geoid postprocessor, which only works in 3D but this postprocessor is of course not limited to 3D so I will update that. Does this need a changelog entry?

Reply:
No need for a changelog entry, it is simply a better way to do the same thing.